main.py

```diff
@@ -13,6 +14,25 @@
 site_data = {}
 by_uid = {}
 
+times = [
```

Comment:
No, I don't want any of this in the python code. It's very hacky and last minute for ICLR.

Reply:
Can you help me by letting me know how do you want this to be implemented?

Reply:
Yeah, ideally this would be a in a separate csv file.